opensrp-chw/src/kk/java/org/smartregister/chw/interactor/AncHomeVisitInteractorFlv.java

```diff
@@ -293,6 +294,18 @@ private void evaluateEarlyStimulation(LinkedHashMap<String, BaseAncHomeVisitActi
         actionList.put(context.getString(R.string.anc_home_visit_early_stimulation), earlyStimulation);
     }
 
+    private void evaluateChwObservation(LinkedHashMap<String, BaseAncHomeVisitAction> actionList,
+                                          Map<String, List<VisitDetail>> details,
+                                          final Context context) throws BaseAncHomeVisitAction.ValidationException {
+        BaseAncHomeVisitAction chw_observations = new BaseAncHomeVisitAction.Builder(context, context.getString(R.string.anc_home_visit_chw_observations))
+                .withOptional(false)
+                .withDetails(details)
+                .withFormName("anc_hv_chw_observations")
+                .withHelper(new ChwObservationsAction())
```

Comment:
Here you need to include the processing mode

`.withProcessingMode(ProcessingMode.COMBINED)`